service.go

```diff
@@ -169,6 +175,10 @@ func (opts *Opts) applyDefaults(ctx context.Context) error {
 		opts.SecretStore = secretStore
 	}
 
+	if opts.P2PRdvpMaddrs == nil {
+		opts.P2PRdvpMaddrs = []string{ipfsutil.DefaultP2PRdvpMaddr}
```

Comment:
You apply this default value here and in mobile.go. Is it better to set it only in mobile.go?

Reply:
We'll leave it, as discussed.